fdbclient/BackupAgentBase.actor.cpp

```diff
@@ -345,6 +344,31 @@ ACTOR static Future<Void> decodeBackupLogValue(Arena* arena,
 				}
 			}
 
+			// Decrypt mutation ref if encrypted
+			if (logValue.isEncrypted()) {
+				encryptedLogValue = logValue;
+				state EncryptCipherDomainId domainId = logValue.encryptionHeader()->cipherTextDetails.encryptDomainId;
+				Reference<AsyncVar<ClientDBInfo> const> dbInfo = cx->clientInfo;
+				try {
+					TextAndHeaderCipherKeys cipherKeys =
+					    wait(getEncryptCipherKeys(dbInfo, *logValue.encryptionHeader(), BlobCipherMetrics::RESTORE));
+					logValue = logValue.decrypt(cipherKeys, tempArena, BlobCipherMetrics::BACKUP);
+				} catch (Error& e) {
+					// It's possible a tenant was deleted and the encrypt key fetch failed
```

Comment:
How about we record the error details regardless of `key_fetch_failed`, decryption failing shouldn't matter otherwise I think